### PR TITLE
Update README to use LLVM LLDB for debugging.

### PR DESCRIPTION
**Pull request by voltur01**

### CODE REVIEW

**Comment by mplatings** [comment]
LGTM

File: samples/README.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ the QEMU Arm System emulator. On Ubuntu Linux it can be installed as follows:
 The Windows installer can be downloaded from
 https://www.qemu.org/download/#windows.
 
-To debug the samples you will need to install GDB provided by the
-[GNU Arm Embedded Toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads).
+To debug the samples you will need to install [LLDB](https://lldb.llvm.org/)
```
[the remark this answers is not in the document]
Comment (voltur01) [reply samples/README.md:36]: Good point, Simon, thanks. Let me tweak it a bit.

[the remark this answers is not in the document]
Comment (voltur01) [reply samples/README.md]: Updated.

**Review by statham-arm — APPROVED** [review]
Thanks, that makes more sense :-)

**Pull request merged**